lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb

```diff
@@ -340,6 +343,16 @@ def local_command_file(filename)
       File.join(ensure_local_directory(local_command_dir), filename)
     end
 
+    def socket_dir
+      File.join(@socket_working_dir, 'sockets', "foreman-ssh-cmd-#{@id}")
+    end
+
+    def socket_file
+      socket_path = File.join(ensure_local_directory(socket_dir), 'socket')
+      raise "Socket path #{socket_path} too long" if socket_path.length > SOCKET_PATH_MAX_LENGTH
```

Comment:
This happens at runtime, doesn't it? It would be better if we could check this at startup and disable the whole feature if we know it cannot work

Reply:
Okay, thanks for suggestion :+1:  I guess it should be validated somewhere in this [file](https://github.com/theforeman/smart_proxy_remote_execution_ssh/blob/master/lib/smart_proxy_remote_execution_ssh.rb), right?

Reply:
Correct